src/Symfony/Component/Inflector/Inflector.php

```diff
@@ -231,6 +231,9 @@ final class Inflector
         // bacteria (bacterium), criteria (criterion), phenomena (phenomenon)
         ['noi', 3, true, true, 'ions'],
 
+        // coupon (coupons)
+        ['nopuoc', 6, true, true, 'coupons'],
```

Comment:
maybe we can widen this to
`['nop', 3, true, true, 'pons'],`

Reply:
I confirm we should do this, see https://en.wiktionary.org/wiki/Category:English_plurals_ending_in_%22-a%22: it lists no English words that end with `pon` and have a `pa` plural.